packages/core/content-releases/server/src/bootstrap.ts

```diff
@@ -57,5 +58,16 @@ export const bootstrap = async ({ strapi }: { strapi: LoadedStrapi }) => {
         }
       },
     });
+
+    if (strapi.features.future.isEnabled('contentReleasesScheduling')) {
+      try {
+        await getService('scheduling', { strapi }).syncFromDatabase();
+      } catch (e) {
+        strapi.log.error(
+          'Error while syncing scheduled jobs from the database in the content-releases plugin. This could lead to errors in the releases scheduling.',
+          e
+        );
+      }
```

Comment:
I believe it would be better to not await this service (if we don't want to prevent strapi starting up on bootsrap that is) but rather:
```suggestion
        getService('scheduling', { strapi }).syncFromDatabase().catch(e =>  {
        strapi.log.error(
          'Error while syncing scheduled jobs from the database in the content-releases plugin. This could lead to errors in the releases scheduling.',
          e
        );
      })
```

Side question, maybe we do want to stop strapi if the scheduling fails 🤔 in that case we should passthrough the error in the catch

Reply:
Hmm 🤔 I think the only scenarios where the sync would fail if that you have something bad with your database or your releases tables. I guess in both cases is okay to stop Strapi because they are major errors

Reply:
I agree. But we need to get the scheduled releases on startup. Otherwise we can't ensure persistence on the scheduling 😞

Reply:
is there a way to destroy strapi on runtime? (that sounds bad)
I mean, crash the app intentionally if this process fail. That way we could remove the `await` and keep it async.

If not lets keep it this way

Reply:
🤔 That sounds like the first behaviour. I mean, don't handle the error so if something fail it would throw the error to the user's console and stop Strapi

Reply:
oh you are right :) , yeah, I would go with that then :) and I would keep the log in it if possible, to know whats broken. wdyt?